core/server/apps/amp/lib/helpers/amp_components.js

```diff
@@ -0,0 +1,29 @@
+// # Amp Components Helper
+// Usage: `{{amp_components}}`
+//
+// Reads through the AMP HTML and adds neccessary scripts
+// for each extended component
+// If more components need to be added, we can add more scripts.
```

Comment:
You caught exactly one tag, that actually needs to be transformed into `amp-audio` or (if the transform in Amperize doesn't exist yet) at least remove the HTML tag, as this is a prohibited tag. This is not implemented yet, so your test reflected exactly the behaviour it should.